README.md

```diff
@@ -110,11 +119,11 @@ yarn test
 yarn run jest --u
 ```
 
-#### clayui.com
+-   #### **Clayui.com**
```

Comment:
I'm not sure you need this to be capitalized since it's an actual URL, and I don't think it needs to be in bold, since you're already using `####`.